uisdk/src/main/java/com/karhoo/uisdk/screen/booking/BookingActivity.kt

```diff
@@ -382,6 +382,7 @@ class BookingActivity : BaseActivity(), AddressBarMVP.Actions, BookingMapMVP.Act
         toolbar.visibility = View.VISIBLE
         navigationDrawerWidget.setDrawerLockMode(DrawerLayout.LOCK_MODE_UNLOCKED)
 
+        journeyDetailsStateViewModel.process(AddressBarViewContract.AddressBarEvent.ResetJourneyDetailsEvent)
```

Comment:
can you please look into the incorrect offset (the pickup icon and the user location) after the user is navigated back to the map